src/script_interface/observables/initialize.cpp

```diff
@@ -137,8 +138,9 @@ void initialize() {
   REGISTER_PID_OBS(ComVelocity);
   REGISTER_PID_OBS(ComForce);
   REGISTER_PID_OBS(ParticleDistances);
-  REGISTER_PID_OBS(ParticleAngles);
+  REGISTER_PID_OBS(BondAngles);
   REGISTER_PID_OBS(ParticleDihedrals);
```

Comment:
Here we might as well rename `ParticleDistances -> BondLengths` and `ParticleDihedrals -> BondDihedrals`

Reply:
I think ```ParticleDistances``` is the correct name. It doesn't need to be a bondlength you want to track.